innovation.game.php

```diff
@@ -12615,6 +12710,29 @@ function stInterInteractionStep() {
                     }
                     break;
 
+                // id 190, Artifacts age 8: Meiji-Mura Stamp Vending Machine
+                case "190N1A":
+                    // Draw and score three cards of the returned card's value.
+                    $age_to_score = self::getGameStateValue('age_last_selected');
+                    self::executeDraw($player_id, $age_to_score, 'score');
+                    self::executeDraw($player_id, $age_to_score, 'score');
+                    self::executeDraw($player_id, $age_to_score, 'score');
+                    break;
```

Comment:
I think you need to put this inside of a n > 0 check, otherwise it won't behave correctly if you don't have a card to return.

Reply:
There isn't an "if you do", so it would result in 0s for the returned card.

Reply:
Great catch!